.pre-commit-config.yaml

```diff
@@ -0,0 +1,55 @@
+default_language_version:
+  python: python3
+# TODO remove _LEGACY once it's gone
+exclude: '\.(tsv|fasta|gb)$|^ingest/vendored/|^_LEGACY'
+repos:
+  - repo: https://github.com/snakemake/snakefmt
+    rev: v0.10.1
+    hooks:
+      - id: snakefmt
+        language_version: python3
```

Comment:
[Ditto](https://github.com/nextstrain/seasonal-cov/pull/10#discussion_r1612043519)
> I think we need to either bump snakefmt to v0.10.2 or pin python to 3.11 to avoid https://github.com/snakemake/snakefmt/issues/227

Reply:
Pushed a correction for this.